x-pack/plugins/apm/public/application/application.test.tsx

```diff
@@ -56,15 +61,24 @@ describe('renderApp', () => {
         },
       },
     };
-    const params = {
+    const appMountParameters = {
       element: document.createElement('div'),
       history: createMemoryHistory(),
       setHeaderActionMenu: () => {},
     };
 
     const data = dataPluginMock.createStartContract();
     const embeddable = embeddablePluginMock.createStartContract();
-    const startDeps = {
+
+    const pluginsStart = ({
+      observability: {
+        navigation: {
+          registerSections: () => registerSectionsSpy,
```

Comment:
Is it really necessary to have `registerSectionsSpy`  as a variable?  It looks like it is not being used anywhere.
```suggestion
          registerSections: jest.fn()
```

Reply:
Good catch. I ended up not using it in any expectations.